src/panels/config/core/ha-config-url-form.ts

```diff
@@ -0,0 +1,170 @@
+import "@material/mwc-button/mwc-button";
+import "@polymer/paper-input/paper-input";
+import type { PaperInputElement } from "@polymer/paper-input/paper-input";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import "../../../components/ha-card";
+import { saveCoreConfig } from "../../../data/core";
+import type { PolymerChangedEvent } from "../../../polymer-types";
+import type { HomeAssistant } from "../../../types";
+
+@customElement("ha-config-url-form")
+class ConfigUrlForm extends LitElement {
+  @property() public hass!: HomeAssistant;
+
+  @property() private _error?: string;
+
+  @property() private _working = false;
+
+  @property() private _external_url!: string;
+
+  @property() private _internal_url!: string;
+
+  protected render(): TemplateResult {
+    const canEdit = ["storage", "default"].includes(
+      this.hass.config.config_source
+    );
+    const disabled = this._working || !canEdit;
+
+    return html`
+      ${this.hass.userData?.showAdvanced
```

Comment:
This check should be on top of render outside the template:
```
If(!advanced) {
 return html``
}
```

Reply:
Right add a guard